frontend_tests/test_happy_mp.py

```diff
@@ -76,24 +80,27 @@ def is_element_present(self, how, what):
 
 
 def test_generator():
-    for scenario in SCENARIOS:
-        for username in USERS:
-            yield run_scenario, scenario, username
+    for username in USERS:
+        yield run_scenario, username, SCENARIOS
 
 
-def run_scenario(scenario, username):
+def run_scenario(username, scenario):
     user = User(username)
     try:
-        globals()[scenario](user)
-    except NoSuchElementException as e:
-        assert False, "Cannot find element {}\n{}".format(e.msg, ''.join(traceback.format_stack()))
+        if isinstance(scenario, str):
+            globals()[scenario](user)
+        if isinstance(scenario, list):
+            for s in scenario:
+                globals()[s](user)
```

Comment:
Do we need to specify them at all? By default we want to run all of them, all the time right?